BusBoard.ConsoleApp/Program.cs

```diff
@@ -3,13 +3,44 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RestSharp;
+using RestSharp.Authenticators;
 
 namespace BusBoard.ConsoleApp
 {
   class Program
   {
     static void Main(string[] args)
     {
+      Console.WriteLine("Please enter your desired stop point ID:");
+      var stopCode = Console.ReadLine();
+      Console.WriteLine(" ");
+      var response = GetListOfArrivalPredictionsForStopPoint(stopCode);
+      var buses = GetNext5Buses(response);
+      Console.WriteLine($"Next 5 Buses at stop {stopCode}");
+      foreach (var bus in buses)
+      {
+        Console.WriteLine($"VehicleID: {bus.VehicleId}, ETA: {bus.ExpectedArrival.AddHours(1).ToString("HH:mm:ss tt")}");
+      }
+      Console.WriteLine(" ");
+    }
+    
+
+    private static IRestResponse GetListOfArrivalPredictionsForStopPoint(string stopCode)
+    {
+      var requestUrl = "https://api.tfl.gov.uk";
+      var client = new RestClient(requestUrl);
+      var request = new RestRequest($"StopPoint/{stopCode}/Arrivals", DataFormat.Json);
+      var response = client.Execute(request);
```

Comment:
You should be able to do `client.Execute<List<Bus>>(request).Data` to get the buses out in a single step.

Reply:
oh thats cool, i didnt manage to find that